src/assets/translations/messages-en.json

```diff
@@ -154,13 +154,25 @@
       "invalidMax": "Invalid Amount. The maximum trade amount for this pair is %{maxLimit}.",
       "insufficientFunds": "Insufficient funds for miner fee and amount",
       "transactionRejected": "User has rejected the transaction",
+      "broadcastFailed": "An error occured while broadcasting the transaction",
       "insufficientFundsForLimit": "Insufficient funds. The minimum trade amount for this pair is %{minLimit}",
       "insufficientFundsForAmount": "Insufficient funds. Your %{symbol} balance is less than the selected amount",
       "noLiquidityError": "Not enough liquidity available",
       "overMaxSlippage": "This trade would result in over %{slippagePercentage}% slippage. Please trade a lower amount for a better price",
       "balanceToLow": "Balance too low. The minimum trade amount for this pair is %{minLimit}",
       "dexTradeFailed": "Trade Failed, Tokens Returned",
-      "quoteFailed": "An error occurred getting a quote for your trade."
+      "quoteFailed": "An error occurred getting a quote for your trade.",
+      "failedQuoteExecuted": "Cannot execute a failed quote",
+      "sellAssetRequired": "Cannot execute a quote without an asset to sell",
```

Comment:
This is on purpose, this wasn't intended to be a 1/1 match with `swapper` errors since `web` is user-facing and higher-level, so IMO the terminology `sellAccountId` makes more sense vs. `sellAssetAccountId`. Happy to change translation keys if anyone has a better naming!